packages/ketcher-core/src/application/render/renderers/constants.ts

```diff
@@ -15,6 +15,7 @@ export const MONOMER_SYMBOLS_IDS = {
     hover: '#sugar-selection',
     selected: '#sugar-selection',
     body: '#sugar',
+    variant: '#sugar-variant',
```

Comment:
Don't we want to use `ambiguous` here to avoid ambiguity (hehe) between `variant` and `ambiguous` in our code?